File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.swift
import Foundation

var allStudents = Set(1...30)

for _ in 1...28 {
    if let input = readLine(), let num = Int(input) {
        allStudents.remove(num)
    }
}

let missing = allStudents.sorted()

for student in missing {
    print(student)
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java

import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        //StringTokenizer st = new StringTokenizer(br.readLine());
        //String input = br.readLine();
        //int a = Integer.parseInt(br.readLine());

        boolean[] std = new boolean[31];  //

        for (int i = 0; i < 28; i++) {
            int num = Integer.parseInt(br.readLine());
            std[num] = true;
        }

        for (int i = 1; i <= 30; i++) {
            if (!std[i]) {
                System.out.println(i);
            }
        }

        br.close();
    }
}
